apps/frontend/src/components/global/groups/GroupModal.vue

```diff
@@ -181,9 +181,13 @@ export default class GroupModal extends Vue {
     }
   }
 
-  async cancel(): Promise<void> {
-    this.dialog = false;
-    location.reload();
+  // Upon group name change, the component will detect whether the current state is acceptable
+  uniqueName = true;
+  checkUniqueName(name: string) {
+    this.uniqueName = !GroupsModule.allGroups.some(
```

Comment:
why are we doing the double negation?  feels like the variable name should just be changed